src/app/checkout/navigateToOrderConfirmation.tsx

```diff
@@ -1,7 +1,14 @@
 import { noop } from 'lodash';
 
-export default function navigateToOrderConfirmation(location = window.location): Promise<never> {
-    const url = `${location.pathname.replace(/\/$/, '')}/order-confirmation`;
+export default function navigateToOrderConfirmation(location = window.location, orderId?: number): Promise<never> {
+    let url: string;
+    const checkoutId = window.location.pathname.replace('/checkout/', '').replace('embedded-checkout', '');
```

Comment:
You can do this `window.location.pathname.split('/').pop()` and that way we don't need to rely on strings in the url
?

Reply:
Thanks for the tip. Thought it would be benificial to have a single method to tell whether it is a Buy Now Cart. So I also modified the code that displays the edit cart button.